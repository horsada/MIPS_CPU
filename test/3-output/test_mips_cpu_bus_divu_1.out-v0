FINAL OUT: 00000002
