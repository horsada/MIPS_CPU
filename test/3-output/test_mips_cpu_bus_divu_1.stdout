WARNING: mips_cpu_ram.v:24: $readmemh: Standard inconsistency, following 1364-2005.
WARNING: mips_cpu_ram.v:24: $readmemh(../test/1-hex/test_mips_cpu_bus_divu_1.hex.txt): Too many words in the file for the requested range [0:65535].
VCD info: dumpfile mips_cpu_bus_tb.vcd opened for output.
Storing LO 00000002 in rd  2
FINAL OUT: 00000002
